### DISCUSSION TITLE
This phone number is already associated with a Vercel account.

### DISCUSSION THREAD

**Discussion by rikakomurakami** [vercel/community#6258] [Account Issue] [answered]
### Question

When I tried to log in to Vercel with my GitHub account, it said "Please enter your phone number to verify your login. You will receive a single use code to submit." When I entered the number, it said "This phone number is already associated. with a Vercel account.'' I can't log in at all.

**Comment by amyegan** [comment, marked as answer]
Hey @rikakomurakami. Please email [registration@vercel.com](mailto:registration@vercel.com) to open a case if you haven't already. I can't safely verify your account information here in public, but that email will get you in touch with our account experts 🙏
